Dockerfile

```diff
@@ -9,41 +9,47 @@ ENV LANG="en_US.UTF-8" \
     SINUS_DIR="/sinusbot" \
     YTDL_BIN="/usr/local/bin/youtube-dl" \
     YTDL_VERSION="latest" \
-    TS3_VERSION="3.0.18.2" \
-    TS3_DL_ADDRESS="http://teamspeak.gameserver.gamed.de/ts3/releases/" \
-    SINUSBOT_DL_URL="https://www.sinusbot.com/dl/sinusbot-beta.tar.bz2"
+    TS3_VERSION="3.1.8" \
+    TS3_DL_ADDRESS="http://dl.4players.de/ts/releases/" \
+    SINUSBOT_DL_URL="https://www.sinusbot.com/dl/sinusbot.current.tar.bz2"
 
 ENV SINUS_DATA="$SINUS_DIR/data" \
     SINUS_DATA_SCRIPTS="$SINUS_DIR/scripts" \
     TS3_DIR="$SINUS_DIR/TeamSpeak3-Client-linux_amd64"
 
 RUN groupadd -g "$SINUS_GROUP" sinusbot && \
-    useradd -u "$SINUS_USER" -g "$SINUS_GROUP" -d "$SINUS_DIR" sinusbot && \
-    apt-get -q update -y && \
+    useradd -u "$SINUS_USER" -g "$SINUS_GROUP" -d "$SINUS_DIR" sinusbot
+    
+RUN apt-get -q update -y && \
     apt-get -q upgrade -y && \
-    apt-get -q install -y x11vnc xvfb libxcursor1 ca-certificates bzip2 \
+    apt-get -q install -y x11vnc xvfb libxcursor1 ca-certificates bzip2 libnss3 libegl1-mesa x11-xkb-utils libasound2 \
         libglib2.0-0 libnss3 locales wget sudo python less && \
     locale-gen --purge "$LANG" && \
     update-locale LANG="$LANG" && \
     echo "LC_ALL=en_US.UTF-8" >> /etc/default/locale && \
     echo "LANG=en_US.UTF-8" >> /etc/default/locale && \
-    update-ca-certificates && \
-    mkdir -p "$SINUS_DIR" && \
+    update-ca-certificates
+    
+RUN mkdir -p "$SINUS_DIR" && \
```

Comment:
Could you please not split the image into multiple layers?
When you split the layers you always need to do "clean up" after each layer to prevent temp data/caches to be "moved" between layers.